src/site/content/en/blog/variable-fonts/index.md

```diff
@@ -7,358 +7,694 @@ description: >
+font. Some instances are named by the font developer, for example Weight axis
+location 600 is called SemiBold.
+
+The variable font [Roboto Flex](https://github.com/TypeNetwork/Roboto-Flex)
+has three styles for its **Weight** axis. The Regular style is at the center,
+and two styles at the opposite ends of the axis, that are lighter and heavier.
+Between these, you can choose from 900 instances:
+
 <figure class="w-figure">
-  <img src="roboto-family.png" alt="Different members of the Roboto family">
-  <figcaption class="w-figcaption">
-    Different fonts within the Roboto family.
+  <img src="roboto-flex-aaaaaa.png" style="max-width: 100%;"
+  alt="The letter 'A' shown in different weights">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Above: Illustrated anatomy of the Weight axis for the typeface Roboto.
   </figcaption>
 </figure>
 
-### Challenges for the Designer and Developer
-
-When a graphic designer prepares their work, they typically export the final
-artwork in a way that either embeds all of the fonts used, or they provide
-an export archive that contains all of the assets used separately, a bit like a
-basic website. The cost to the designer depends on the suitability of the
-typeface for the rendering context  (is it legible at small sizes, for example)
-and the license to use the fonts.
-
-On the web, we have to consider both aspects, plus the associated bandwidth
-costs. For every member of a typeface family used in our designs, we have had to
-require another font file to be downloaded by our users before they can see that
-text. Only including the Regular and Bold styles, plus their italic
-counterparts, can amount to 500&nbsp;KB or more of data. This has been a sticking point
-for web designers and developers, as a richer typographic experience comes
-at a cost. This is even before we have dealt with how the fonts are rendered, and the
-fallback or delayed-loading patterns we are going to use to prevent [FOIT and
-FOUT](https://css-tricks.com/fout-foit-foft/).
-
-## Anatomy of a variable font
-
-A variable font is a collection of master styles, with one central "default"
-master (usually the Regular font style) and multiple registered "axes" which
-tie the central master to the other masters. For example, the **Weight** axis
-might connect a Light style master to the default style and through to the Bold
-style master. The individual styles that can be located along this axis are
-called instances.
-
-For example, the variable font [Amstelvar](https://github.com/TypeNetwork/Amstelvar)
-has three masters for its **Weight** axis: The Regular master is at the center,
-and two masters, Thinner and Bolder, are at the opposite ends of the axis.
-Between these there are potentially 200 instances that can be chosen by the
-designer or developer:
+The font developer can offer a set of different axes, and because they all share
+the same default styles, you can combine them. Roboto has three styles in a
```

Comment:
```suggestion
The font developer can offer a set of different axes. You can combine them
because they all share the same default styles. Roboto has three styles in a
```